src/plots/cartesian/dragbox.js

```diff
@@ -434,9 +434,14 @@ module.exports = function dragBox(gd, plotinfo, x, y, w, h, ns, ew) {
 
     // everything but the corners gets wheel zoom
     if(ns.length * ew.length !== 1) {
-        // still seems to be some confusion about onwheel vs onmousewheel...
-        if(dragger.onwheel !== undefined) dragger.onwheel = zoomWheel;
-        else if(dragger.onmousewheel !== undefined) dragger.onmousewheel = zoomWheel;
```

Comment:
Can we keep that `onmousewheel` handler in case we need it in older browsers?

Reply:
I think the only difference between what @dfcreative has here and what we had before is that now we *always* attach to either `wheel` or `mousewheel`, whereas before we could potentially attach to neither if the browser doesn't support either one (I had to play with this - you may already know this but what's going on is if the browser supports `wheel`, then `element.onwheel = null` before any handler is attached, not `undefined`). And even if there is a case where the browser really doesn't support either, there shouldn't be any problem attaching to a nonexistent event.

Reply:
Oh I see. My bad. :ok_hand: